en/docs/SetupAndInstall/UpgradingWSO2APIManager/upgrading-from-250-to-300.md

```diff
@@ -3,350 +3,1676 @@
     -   Move all your tenant Synapse configurations.
-        Copy the contents in the `<CURRENT_API-M_HOME>/repository/tenants` directory and replace the contents in the `<API-M_2.6.0_HOME>/repository/tenants` directory with the copied contents.
+        Copy the contents in the `<OLD_API-M_HOME>/repository/tenants` directory and replace the contents in the `<API-M_3.0.0_HOME>/repository/tenants` directory with the copied contents.
 
-#### Step 1.2 - Optionally, migrate the configurations for WSO2 API-M Analytics
+    !!! warning
+        When moving the Synapse configurations, **do not replace** the following set of files as they contain some modifications in API-M 3.0.0 version.
 
-!!! warning
-This step is **only required** if you have WSO2 API-M-Analytics configured in your current deployment.
-Follow the steps below to migrate APIM Analytics 2.5.0 to APIM Analytics 2.6.0
+        -   \_RevokeAPI_.xml
+        -   \_cors_request_handler_.xml
+        -   main.xml
 
-##### Step 1.2.1 - Configure WSO2 API-M Analytics 2.6.0
+7. If you manually added any custom OSGI bundles to the `<API-M_2.5.0_HOME>/repository/components/dropins` directory, copy those to the `<API-M_3.0.0_HOME>/repository/components/dropins` directory. 
 
-1.  Download WSO2 API Manager Analytics 2.6.0 from [here](http://wso2.com/api-management/) .
-2.  Create a new database for the new statistics database and configure it in the `<APIM_ANALYTICS_2.6.0_HOME>/conf/worker/deployment.yaml` as follows:
-    In this example shows to configure MySQL database configurations.
+8. If you manually added any JAR files to the `<API-M_2.5.0_HOME>/repository/components/lib` directory, copy those and paste them in the `<API-M_3.0.0_HOME>/repository/components/lib` directory.
 
-    ``` java
-        - name: APIM_ANALYTICS_DB
-              description: "The datasource used for APIM statistics aggregated data."
-              jndiConfig:
-                name: jdbc/APIM_ANALYTICS_DB
-              definition:
-                type: RDBMS
-                configuration:
-                  jdbcUrl: 'jdbc:mysql://localhost:3306/spDatabase?autoReconnect=true'
-                  username: username
-                  password: password
-                  driverClassName: com.mysql.jdbc.Driver
-                  maxPoolSize: 50
-                  idleTimeout: 60000
-                  connectionTestQuery: SELECT 1
-                  validationTimeout: 30000
-                  isAutoCommit: false
-    ```
+9. Log4j version has been upgraded to log4j2 in API Manager 3.0.0. You will notice that there is a log4j2.properties file in the `<API-M_3.0.0_HOME>/repository/conf/` directory instead of the log4j.properties file. Follow [Upgrading to Log4j2](../UpgradingWSO2APIManager/upgrading-to-log4j2.md) to migrate your existing log4j configurations.
```

Comment:
Shall we change like this?

WSO2 API Manager 3.0.0 has been upgraded to log4j2 (from log4j).

Reply:
Changed